### ISSUE TITLE
Add focus note for Styled File Input Button

### ISSUE THREAD

**Issue by scottaohara** [Chalarangelo/mini.css#65]
The styled input button should have an additional note pertaining to the fact that they do not & can not have a keyboard focus state, without the use of JavaScript.

**Comment by Chalarangelo** [comment]
Noted, I will be adding this in **v2.1.1**, thanks for the suggestion!
